spec/javascript/components/lease_ups/__snapshots__/LeaseUpsPage.test.js.snap

```diff
@@ -1,10933 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
these tests weren't actually testing anything. the snapshots were just the empty lease up applications table